packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -122,6 +121,7 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
             enableSmoothScroll: true,
             overscan: 3,
             isScrollingResetDelay: 250,
+            useScrollendEvent: false,
```

Comment:
@jattasNI can you verify this change does not introduce any regression in the table scroll behavior like shown here https://github.com/ni/nimble/pull/2361#discussion_r1742379597

Reply:
@rajsite I tried macOS Safari, Firefox, and Edge. I didn't see any issues with table scrolling.

Reminder to resolve #2382 when this PR completes.